### PR TITLE
Added info on standard gamepads for HTML5

### PR DESCRIPTION
**Pull request by britzl**

### CODE REVIEW

File: docs/en/manuals/input-gamepads.md [hunk 1]
```diff
@@ -149,6 +147,14 @@ If your game is running from inside an `iframe` you must also make sure that the
 <iframe allow="gamepad"></iframe>
 ```
 
+### Standard gamepad
+(From Defold 1.3.8)
```
Comment (AGulev) [inline docs/en/manuals/input-gamepads.md:151]: 1.3.8?

Comment (britzl) [reply docs/en/manuals/input-gamepads.md:151]: Erm, I forgot that we upped the version!

Comment (AGulev) [reply docs/en/manuals/input-gamepads.md]: But do you really want to release it in current beta?

Comment (britzl) [reply docs/en/manuals/input-gamepads.md]: Oh my god, I am so confused. It should be in the next release!

Comment (britzl) [reply docs/en/manuals/input-gamepads.md]: 1.4.1

**Pull request merged**